dashboards/google-anthos/anthos-clsuter-on-vmware-vm-status.json

```diff
@@ -0,0 +1,107 @@
+{
+  "displayName": "Anthos Cluster on VMware VM Status",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "CPU Readiness Per vCPU (Percentage)",
+        "xyChart": {
+          "dataSets": [
+            {
+              "timeSeriesQuery": {
+                "timeSeriesQueryLanguage": "{ fetch k8s_pod::kubernetes.io/anthos/vm_cpu_ready_seconds | align rate(1m) | add[cluster_name: if(re_full_match(resource.namespace_name, 'kube-system'), resource.cluster_name, resource.namespace_name), ]| every 1m | group_by [resource.project_id, cluster_name, resource.location, metric.node], [value_vm_cpu_ready_seconds_aggregate: aggregate(value.vm_cpu_ready_seconds)]; fetch k8s_container::kubernetes.io/anthos/kube_node_status_capacity | filter (metric.resource == 'cpu') | every 1m | add[cluster_name: resource.cluster_name ] | group_by [resource.project_id, resource.location, cluster_name, metric.node], [value_kube_node_status_capacity_cpu_cores_aggregate: aggregate(value.kube_node_status_capacity)] } | join | div | mul (100)"
```

Comment:
Formatting nit: can you click "Format" in the dashboard builder query editor to prettyprint this MQL chunk? Doesn't matter for dashboard viewers, but if someone copies and edits the dashboard, this will make it easier to read the query itself.

<img width="350" alt="Screenshot 2023-03-13 at 4 13 34 PM" src="https://user-images.githubusercontent.com/6540967/224821929-312d07d8-9436-460b-bfdd-337f58f2c4a5.png">

Same goes for the other 3 big MQL queries here.